test/test_ddl.py

```diff
@@ -162,3 +162,16 @@ def test_afregistrering_sagseventinfo(firedb: FireDb, sagsevent: Sagsevent):
 
     assert si1.registreringtil == si2.registreringfra
     assert si2.registreringtil is None
+
+
+def test_timestamps(firedb: FireDb, koordinat: Koordinat):
+    """
+    Test at TIMESTAMPs kan opløse mikrosekunder.
+    """
+
+    timestamp = dt.datetime(2020, 9, 22, 13, 37, 12, 345)
+    koordinat.t = timestamp
+    firedb.session.add(koordinat)
+    firedb.session.commit()
+
+    assert koordinat.t == timestamp
```

Comment:
Tester dette virkelig det korrekte? koordinat.t er et direkte kig ned i det committede i databasen, og ikke bare den samme gamle værdi?

Reply:
Ja, det gør det. Kan testes ved at kigge på objektid eller andet der sættes af databasen og ikke af koden. Der foregår meget magi i SQLAlchemy men det virker.

Reply:
Det troede jeg (efter diverse forsøg) men lidt søgning rundt omkring antyder at det nok ikke er sandt. Jeg tror det er nok at køre et `firedb.session.refresh()` på objektet inden værdierne i det testes.

Reply:
Ifølge https://www.michaelcho.me/article/sqlalchemy-commit-flush-expire-refresh-merge-whats-the-difference skulle et kald til `commit()` resultere i at et objekt markeres som "expired". Når attributter fra et expired objekt efterspørges laves opslag i databasen. Så alt burde være fint her. Jeg har dog for en sikkerheds skyld indsat et `firedb.session.refresh(koordinat)`.